content/en/monitors/types/cloud_cost.md

```diff
@@ -43,16 +43,40 @@ Optionally, click the **+ Create Monitor** button in the cost report side panel.
 
 ## Select a cost monitor type
 
-Choose between a **Compare Costs Over Time** or a **Set Daily Cost Threshold** monitor type.
+You can select from the following monitor types.
+
+{{< tabs >}}
+{{% tab "Changes" %}}
 
 | Cost Type | Description | Usage Examples |
 | ---  | ----------- | ----------- |
 | Cost Changes  | Compare costs on a daily, weekly or monthly basis | Alert when the difference between today's cost and the week before is over 5% |
```

Comment:
Add periods after the sentences in this table for consistency with your new ones?

Reply:
```suggestion
| Cost Changes  | Compare costs on a daily, weekly or monthly basis | Alert when the difference between today's cost and the week before is over 5%. |
```